docs/blog/2019-12-09-inferring-from-history/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Inferring from history to reduce unnecessary user interaction in Gatsby Cloud
+date: 2019-12-09
+author: Shannon Soper
+tags: ["ux", "cloud"]
+---
+
+In my last blogpost, I introduced how we’re [making it easier to start with Gatsby Cloud for 
+free](https://www.gatsbyjs.org/blog/2019-11-25-getting-started-with-gatsby-cloud/) with starters. This blogpost introduces the next set of changes that we shipped to help users get started faster. How did we do it? Inferring from history to reduce unnecessary user interaction. Quite a mouthful and needs some explanation.
+
+## The problem
+So what problem did we learn about?
+
+When watching folks onboard onto Gatsby Cloud, there was a clear point of confusion. This screenshot depicts step two of Gatsby Cloud onboarding, during which users select an organization from GitHub. I drew a speech bubble on top of the screenshot pointing out that users are confused by this screen. The speech bubble says: “Wait, I don't want to create a repository in my work's GitHub organization. Terminate mission!"
+ 
+![Screenshot](select-work-org-confusion.png)
```

Comment:
Since you've described the image immediately before, I agree that it doesn't need to be repeated in alt text. What do you think about using a phrase like this? (Marcy suggested "Create a new site".) 

Otherwise, I'd consider leaving the alt text blank. I would expect doing so to result in an empty `alt` attribute as opposed to leaving it out entirely. Right now, what actually happens is that we fill it in using the image's name. 
```suggestion
![Selecting an organization](select-work-org-confusion.png)
```

Reply:
Thanks! I thought of a third option and would appreciate your thoughts @AishaBlake: a third option is to take the explanatory text from the previous paragraph and place it in the alt text like so  I was experimenting with doing the explanation as part of the paragraph text because I realized after spending a lot of time on the alt text initially that I think every reader could actually benefit from the explanation. I could be wrong about that...should I put that longer explanation in the alt text instead?

Reply:
Fixed! Thank you!